### PR TITLE
Extending SunInfo to support latitude and longitude

### PR DESCRIPTION
**Pull request by mintsoft**
Addresses https://github.com/duckduckgo/zeroclickinfo-goodies/issues/901

Adds support for "at latN lonE" in decimal and arcseconds as requested by @rpadovani

### CODE REVIEW

**Comment by jagtalon** [comment]
Nice!

**Comment by mintsoft** [comment]
image
image

**Comment by mintsoft** [comment]
ping @jagtalon @moollaza

**Comment by mintsoft** [comment]
also @MrChrisW

File: lib/DDG/Goodie/SunInfo.pm [hunk 1]
```diff
@@ -34,22 +35,41 @@ my $sunset_svg = goodie_img_tag({
...
+    
     return unless $dt;                                  # Also going to need to know which day.
-    $dt->set_time_zone($tz);
-
+    $dt->set_time_zone($tz) unless ($+{'lat'} && $+{'lon'});
+    
+    $lon = parse_arc($+{'lon'}) if ($+{'lon'});
+    $lat = parse_arc($+{'lat'}) if ($+{'lat'});
+    
+    $where = "coordinates ${lat}°N ${lon}°E" if($+{'lat'} && $+{'lon'});
```
Comment (jagtalon) [inline lib/DDG/Goodie/SunInfo.pm]: Was thinking maybe it should start with a capital letter, i.e., "Coordinates ..."

**Comment by mintsoft** [comment]
@jagtalon should be done now

**Comment by jagtalon** [comment]
:+1:

**Comment by jagtalon** [comment]
:+1:

**Comment by jagtalon** [comment]
@mintsoft it's up! https://duckduckgo.com/?q=sunset+at+1S+1W&ia=answer

**Comment by mintsoft** [comment]
Awesome thanks for the heads up @jagtalon

**Pull request merged**